core/src/test/scala/unit/kafka/server/DeleteTopicsRequestTest.scala

```diff
@@ -73,7 +73,7 @@ class DeleteTopicsRequestTest extends BaseRequestTest {
 
     // Timeout
     TestUtils.createTopic(zkUtils, timeoutTopic, 5, 2, servers)
-    validateErrorDeleteTopicRequests(new DeleteTopicsRequest(Set(timeoutTopic).asJava, 1),
+    validateErrorDeleteTopicRequests(new DeleteTopicsRequest(Set(timeoutTopic).asJava, 0),
```

Comment:
Yes indeed. Just a short note explaining that we need a timeout of 0 because even 1 can succeed sometimes.